### ISSUE TITLE
No visualization 

### ISSUE THREAD

**Issue by yoanngoma** [ladybug-tools/butterfly#245]
Hi,

I am trying to run butterfly on the outdoor file you provide, but for some reason once I get to the solution part I cannot visualize any solutions. The results seems to converge and the virtual docker seems to work fine, but once I run the solution it just doesn't show anything.

I am quite new to grasshopper so that might have something to do with it, but I didn't really change any of the parameters, so I thought I would be able to properly visualize the air flow as well as the residual evolution. If by any chanche you happen to have any imput on this it would be great.

**Comment by mostaphaRoudsari** [comment]
@yoanngoma can you provide us a screenshot and connect some panels to the outputs?

Also did you watch [the YouTube videos](https://www.youtube.com/playlist?list=PLkjfDmSc5Orx2qJivQBA9Rgk2KAOiESmo)?

**Comment by yoanngoma** [comment]
@mostaphaRoudsari I haven't seen the videos yet, didn't even know there were any. I will watch them and then let you know if I still have questions.  Thank you anyway

**Comment by mostaphaRoudsari** [comment]
Good point! I added the videos on top of the wiki pages. I close this for now. Feel free to re-open or create a new issue if you couldn't solve the problem.

**Comment by yoanngoma** [comment]
Hi @mostaphaRoudsari,

So I watched your video, but it didn't really solve my problem. Here are the differences I could tell between when you ran it and whe I do.

1) First of all, I am studying 29 buildings (colored in the first picture). I imagine that's one of the reasons it makes the process slower when running snappymesh on the docker. Either way, as you can see, my buildings don't really have the same mesh density as yours (although I believe that's just a problem with my programm's visualization).

![1](https://user-images.githubusercontent.com/29441191/27160288-a9844218-5138-11e7-97b4-2e7655955e1e.png)
![2](https://user-images.githubusercontent.com/29441191/27160290-aded84ae-5138-11e7-9c94-7abf5d8687b6.png)

2) The points for the probes didn't appear for me (again, I just think it is a problem of simple visualization). 
![3](https://user-images.githubusercontent.com/29441191/27160417-c004eaf0-5139-11e7-8635-9b5a170d2b7a.png)

3) Finally, and most importantly, when I run the solution, not only it didn't show up on my Rhino visual, but it went much faster thatn it did for your video, when for the snappymesh it was much slower in my computer. 
Here is an error given when I tried to see the residual values:

![4](https://user-images.githubusercontent.com/29441191/27160448-081b90d2-513a-11e7-9109-36379c67342a.png)


Please let me know if you weren't able to see the images and thank you very much for your help.

**Comment by AmyKoshy** [comment]
Hey, i am also not able to see a live visualization on my simulation

![image](https://user-images.githubusercontent.com/30070811/28154616-11f5bf6a-67ab-11e7-94e6-1d9f6128ac1f.png)

**Comment by AmyKoshy** [comment]
oh i just realized there is an error

**Comment by mostaphaRoudsari** [comment]
@AmyKoshy open a new issue for your problem. Your problem doesn't seem to be the same.

**Comment by mostaphaRoudsari** [comment]
Any updates on this issue? Should we close it?

**Comment by AmyKoshy** [comment]
thank you ! i wont be able to check it for while, please close. much appreciated!


________________________________
